terraform/india/development/main.tf

```diff
@@ -82,38 +96,38 @@ module "npw_consumer_ecmwf_ecs" {
   ]
 }
 
+*/
+
 # 4.0
 module "airflow" {
   source                         = "../../modules/services/airflow"
-  environment                    = local.environment
-  vpc_id                         = module.network.vpc_id
-  subnet_id                      = module.network.public_subnet_ids[0]
-  db_url                         = "not-set"
+  aws-environment                = local.environment
+  aws-vpc_id                     = module.network.vpc_id
+  aws-subnet_id                  = module.network.public_subnet_ids[0]
+  rds-db_secret_url              = module.postgres-rds.secret-url
```

Comment:
Just say in the UK platform I actually put airflow into a seperate database (on the same database server). This is because airflow makes about 10 tables, and I think it was good to split up the airflow ones and the ones we made. 

Keep it like this for the moment, and we can always move airflow out later

Reply:
Ah so I see - is it enough to add the extra "/airflow" at the end of the connection url address?

Reply:
New PR to use it's own db here https://github.com/openclimatefix/ocf-infrastructure/pull/393